skyportal/models/group.py

```diff
@@ -365,6 +365,15 @@ class Group(Base):
         doc='The MMADetector spectra visible to this group.',
     )
 
+    mmadetector_time_intervals = relationship(
+        "MMADetectorTimeInterval",
+        secondary="group_mmadetector_time_intervals",
+        back_populates="groups",
+        cascade="save-update, merge, refresh-expire, expunge",
+        passive_deletes=True,
+        doc='The MMADetector time intervals visible to this group.',
+    )
```

Comment:
Just wondering, do we really need to associate a time interval with a group? Is there expected to be permissioning on time intervals?

Reply:
I am kind of envisioning that one might want to post particular data subsets (i.e. detections or something), that might have restrictive access.